bigquery/pandas-gbq-migration/requirements.txt

```diff
@@ -7,6 +7,6 @@ pandas-gbq==0.19.2; python_version > '3.6'
 # pandas-gbq==0.14.1 is the latest compatible version for Python 3.6
 pandas-gbq==0.14.1; python_version < '3.7'
 grpcio==1.56.0
-pyarrow==13.0.0; python_version > '3.6'
+pyarrow==14.0.0; python_version > '3.6'
 # pyarrow==6.0.1 is the latest compatible version for pandas-gbq 0.14.1
-pyarrow==6.0.1; python_version < '3.7'
+pyarrow==14.0.0; python_version < '3.7'
```

Comment:
@shollyman I wonder if this line should be reverted?

Reply:
After some offline discussion, the whole line 12 might be able to be removed, because python < 3.7 are no longer supported in python-bigquery library. @chalmerlowe do you have more context on this?

Reply:
This change seems to be added in [this commit](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/9056/commits/ec23c05af3523e0547d851570d7bb4a2a6da469c) in order to accommodate python version < 3.6. So I will keep this line and revert the change by the dependency bot.